src/main/java/com/ericsson/eiffelcommons/jenkins/JenkinsManager.java

```diff
@@ -355,7 +342,8 @@ public boolean installPlugin(String plugin, String version) throws Exception {
             throw new Exception("A version must be speciified for the Jenkins Plugin.");
         }
 
-        String scriptData = "<jenkins><install plugin='" + plugin + "@" + version + "' /></jenkins>";
+        String scriptData = "<jenkins><install plugin='" + plugin + "@" + version
```

Comment:
Please use String.format :) and possible extract this part to ia method.

Reply:
Done